primitives/state-machine/Cargo.toml

```diff
@@ -22,6 +22,7 @@ codec = { package = "parity-scale-codec", version = "1.2.0" }
 num-traits = "0.2.8"
 rand = "0.7.2"
 sp-externalities = { version = "0.8.0-alpha.2", path = "../externalities" }
+futures = "0.3.1"
```

Comment:
As far as I can see, the dependency is only used in tests? 
And you only need it to create a dummy executor, can we not use the single threaded executor you provide in `sp-core`?